modulegen/internal/dependabot/main.go

```diff
@@ -0,0 +1,26 @@
+package dependabot
+
+func UpdateConfig(configFile string, directory string, packageEcosystem string) error {
+	config, err := readConfig(configFile)
+	if err != nil {
+		return err
+	}
+	config.addUpdate(newUpdate(directory, packageEcosystem))
+	return writeConfig(configFile, config)
+}
+
+func GetUpdates(configFile string) (Updates, error) {
+	config, err := readConfig(configFile)
+	if err != nil {
+		return nil, err
+	}
+	return config.Updates, nil
+}
+
+func CopyDependabotConfig(configFile string, tmpFile string) error {
```

Comment:
The package namespace already provides the context :)
```suggestion
func CopyConfig(configFile string, tmpFile string) error {
```